docs/capis/xnvme.rst

```diff
@@ -205,6 +205,22 @@ xnvme_buf_free
 
 .. doxygenfunction:: xnvme_buf_free
 
+   
+.. _sec-c-api-xnvme-func-xnvme_buf_map:
+
+xnvme_buf_map
+---------------
```

Comment:
For the public API, to completely separate the notion of command buffer, usually a single allocated region of memory, against that of "mapped memory", which might be larger ranges of memory, used by multiple buffers. Then, let's provide a API seperate from buffers. Let's introduce: ``xnvme_mem`` with the methods map/unmap:

- [ ] Add memory-mapping API function ``xnvme_mem_map()``
- [ ] Add memory-mapping API function ``xnvme_mem_unmap()``